src/buildHTML.js

```diff
@@ -1228,7 +1230,7 @@ function sizingGroup(value, options, baseOptions) {
         inner[i].depth *= multiplier;
     }
 
-    return buildCommon.makeFragment(inner);
+    return buildCommon.makeSpan(null, inner, null, attributes);
```

Comment:
This will change the spacing between atoms at sizing group boundaries.  This is because the spacing styles are based on sibling relationships.  If the start of a sizing group is one type of atom and the end is another type of atom we can't apply the existing classes and have the spacing on both ends match.  I think it makes sense to give sizing groups an element in the HTML.  There might be nodes within the LaTeX parse tree which don't have nodes in the HTML.  I think this is okay, but you'll need ways to get this information, e.g. functions to get previous/next sizing command and possible build selections of other nodes based on the location of sizing commands.

Reply:
All changes of `makeFragment` to `makeSpan` will likely have similar results on the spacing of atoms.

Reply:
I think sizing and color could be treated somehow sane in an editor, but `mod` would be bigger problem because it changes structure and semantics. I've reverted chages for former fragments. Maybe I would be able to say more about it later when we have an actual editor.